cloud/amazon/ec2_elb_lb.py

```diff
@@ -698,6 +710,28 @@ def _set_cross_az_load_balancing(self):
         self.elb_conn.modify_lb_attribute(self.name, 'CrossZoneLoadBalancing',
                                           attributes.cross_zone_load_balancing.enabled)
 
+    def _set_access_log(self):
+        attributes = self.elb.get_attributes()
+        if self.access_logs is not None:
+            access_log = boto.ec2.elb.attributes.AccessLogAttribute()
+            access_log.enabled = True
+            access_log.s3_bucket_prefix = ''
+            access_log.emit_interval = 60
+            if 's3_location' not in self.access_logs:
+              self.module.fail_json(msg='s3_location information required')
+            access_log.s3_bucket_name = self.access_logs['s3_location']
+            if 's3_prefix' in self.access_logs:
+              access_log.s3_bucket_prefix = self.access_logs['s3_prefix']
+            if 'interval' in self.access_logs:
+              access_log.emit_interval = self.access_logs['interval']
+            if str(access_log) != str(attributes.access_log):
```

Comment:
Coercing these into a string and doing a comparison is almost certainly not what we want to do here. Is there a reason we can't just do `access_log == attributes.access_log'?

If those don't compare for some reason there is probably a better type we can coerce these into for comparison's sake, e.g. a dict.

Alternatively you could keep a `changed` boolean initialized to `False` and set it to `True` any of the values are set.